pegasus/sites.v3/code.org/public/educate/csp/widgets.haml

```diff
@@ -121,20 +121,12 @@ video_player: true
     %ul.tool-links
       %li
         %i.fa.fa-file-text
-        %a{href: CDO.code_org_url('/curriculum/cspunit2/13/Teacher'), target:'_blank'}
-          Caesar Cipher Lesson plan
-      %li        
-        %i.fa.fa-external-link-square
-        %a{href: CDO.studio_url('/s/cspunit2/stage/13/puzzle/1'), target:'_blank'}
-          Caesar Cipher Stage in CS Principles
-      %li
```

Comment:
Do we not have corresponding links to the Caesar Cipher and Frequency Analysis Units?

Reply:
We're no longer using the Frequency Analysis part, so GT just wanted me to drop those. The new links point to the Caesar Cipher widget, but now that there's just one set of links in that section we can make the link text the same as the other units.